### PR TITLE
Support custom issue report links

### PR DESCRIPTION
**Pull request by feedmeapples**
per user request:
Allows to customize the issue report link through config.yml file

``` yaml
routing:
  default_to_namespace: # my_namespace
  issue_report_link: https://github.com/temporalio/web/issues/new/choose
```

image

### CODE REVIEW

**Comment by sw-yx** [comment]
hmm i want to push back on this/find another solution. what does this user actually want to do? what do they want to link to instead?

**Comment by feedmeapples** [comment]
> hmm i want to push back on this/find another solution. what does this user actually want to do? what do they want to link to instead?
> 
> counter proposed solution: a page on the docs that lets people figure out where to file issues for support/feedback

they wanted to point internal users to custom support forum

**Review by sw-yx — APPROVED** [review]

File: server/config.yml [hunk 1]
```diff
@@ -13,4 +13,5 @@ auth:
 # for more info see docs: https://github.com/temporalio/web#configuring-authentication-optional
 
 routing:
-  default_to_namespace: # my_namespace
+  default_to_namespace: # set this field if your default namespace isn't named 'default'
```
Comment (feedmeapples) [inline server/config.yml:16]:
```suggestion
  default_to_namespace: # internal use only
```

**Comment by jeffschoner-stripe** [comment]
> hmm i want to push back on this/find another solution. what does this user actually want to do? what do they want to link to instead?
> 
> counter proposed solution: a page on the docs that lets people figure out where to file issues for support/feedback

I wanted to add more context since I'm the user who asked @rylandg for this feature. We plan to run temporal-web for use by multiple developers across different teams within our company. If they have questions or see issues with their workflows, it may be about our particular infrastructure rather than Temporal web UI or server. It also might not be obvious to them that they're creating a public Github issue since it links from an internal site and we use GHE which looks similar. In those cases, we'd rather they engage with our team first.

We'll set this to link to a channel on our Slack to start with but eventually we plan to link to a form to open a ticket on our team internally.

**Comment by sw-yx** [comment]
hey Jeff, yeah Ruslan explained to me internally :) all good here

**Pull request merged**